lib/commands/push.ts

```diff
@@ -416,4 +482,17 @@ export default class PushCmd extends Command {
 
 		return selected.extra;
 	}
+
+	checkInvalidOptions(
+		invalidOptions: string[],
+		options: FlagsDef,
+		errorMessage: string,
+	) {
+		invalidOptions.forEach((opt) => {
+			// @ts-ignore : Not sure why typescript wont let me do this?
```

Comment:
I believe this `@ts-ignore` line can be deleted, if the following type info is added to the relevant places:
```
				const localOnlyOptions: Array<keyof FlagsDef> = [
					'nolive',
					'service',
					'system',
					'env',
				];

				const remoteOnlyOptions: Array<keyof FlagsDef> = [
					'release-tag-key',
					'release-tag-value',
				];

	checkInvalidOptions(
		invalidOptions: Array<keyof FlagsDef>,
		...
```